tests/MongoDB.Driver.Tests/Specifications/crud/DropCollectionTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using MongoDB.Bson;
+
+namespace MongoDB.Driver.Tests.Specifications.crud
+{
+    public class DropCollectionTest : CrudOperationTestBase
+    {
+        protected override bool TrySetArgument(string name, BsonValue value)
```

Comment:
Can be virtual?

Reply:
Not sure why you mean. It *is* virtual.

Reply:
It's `abstract` now. In case if it's only `virtual` with default code like `return false`, this overload will not be required.